### DISCUSSION TITLE
Refund for video-downloadhelper

### DISCUSSION THREAD

**Discussion by Ihebbel11** [aclap-dev/video-downloadhelper#1023] [Help and Q&A]
### Your question

Hello , I recently bought video-downloadhelper license but i had quite some problems with it as some videos cannot be downloaded and some other can be downloaded but when they are and i play the videos they are missing some parts thus I kindly request a refund , Thanks 

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: fr
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"5fbb1ee551574d1f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "download_directory": "C:\\Users\\ihebb\\Downloads\\Video\\3-content creation + ai\\talk to camera",
    "never_show_no_incognito_msg_again": true,
    "theme": "system",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 46
}

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by Ihebbel11** [reply]
still isn't available

**Reply by jcfrog** [reply]
Sorry it may take up to 10 business days.

**Reply by Ihebbel11** [reply]
can you check from your side whether it is pending or not ?

**Reply by jcfrog** [reply]
I confirm. Stripe message on our side:
_Successfully refunded due to customer request. It may take a few days for the money to reach the customer's bank account._

**Reply by Ihebbel11** [reply]
can you please tell me when this stripe statement was placed ?

**Comment by jcfrog** [comment]
I found you actually already asked for this refund in June. Stripe says 29 Jun 2024, 09:51 for the refund.

**Comment by shalongasen** [comment]
Hi, yesterday I bought downloadhelper but already bought IDM licence, the app does the same job. I'm requesting a refund because I won't be using downloadhelper. My email handlordsproductions@gmail.com

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.
